src/serializer/ResidualHeapVisitor.js

```diff
@@ -1223,21 +1223,6 @@ export class ResidualHeapVisitor {
     this.visitGenerator(this.globalGenerator);
     for (let moduleValue of this.modules.initializedModules.values()) this.visitValue(moduleValue);
 
-    if (this.realm.react.enabled) {
-      let fixpoint_rerun = () => {
```

Comment:
Why were we using this `fixpoint_rerun` recursion? I moved the visiting logic into `ResidualReactElementVisitor` along with the other visiting logic for that class and didn’t observe any test failures.

Reply:
We may have been doing this in the case where React is behind something that is behind a value that is delayed. If you run your PR on our internal bundle that I sent you the other night, do we hit any issues when compiling it?

Reply:
Internal bundle builds 👍